packages/manager/modules/manager-pci-common/src/components/quantity-selector/QuantitySelector.component.tsx

```diff
@@ -48,116 +35,64 @@ export function QuantitySelector({
   contentClassName,
 }: Readonly<QuantitySelectorProps>) {
   const { t } = useTranslation('pci-quantity-selector');
-  const [, setCounter] = useState(0);
   const error = {
     min: min !== undefined && value < min,
     max: max !== undefined && value > max,
     nan: Number.isNaN(value),
   };
-  const isValid = !error.min && !error.max && !error.nan;
-  const forceRedraw = () => setCounter((i) => i + 1);
-
-  // forced redraw to fix ODS quantity not refreshing
-  useEffect(forceRedraw, [min, max]);
 
   return (
-    <OsdsFormField className={className} inline>
+    <OdsFormField className={className}>
       <div className={contentClassName}>
         <div slot="label" className="flex gap-2">
-          {label && (
-            <OsdsText
-              color={ODS_THEME_COLOR_INTENT.text}
-              size={ODS_THEME_TYPOGRAPHY_SIZE._200}
-            >
-              {label}
-            </OsdsText>
-          )}
+          {label && <OdsText preset="span">{label}</OdsText>}
```

Comment:
Previous text had `ODS_THEME_TYPOGRAPHY_SIZE._200` size that means `14px` no?
If so, we need to add a custom tailwind style because `span` preset has a `16px` font size by default